pkg/api/v1/pod/util.go

```diff
@@ -297,58 +298,6 @@ func IsPodReadyConditionTrue(status v1.PodStatus) bool {
 // GetPodReadyCondition extracts the pod ready condition from the given status and returns that.
 // Returns nil if the condition is not present.
 func GetPodReadyCondition(status v1.PodStatus) *v1.PodCondition {
```

Comment:
it might be worth it to move this function to corev1helpers as well, since it is just a wrapper for `GetPodCondition`

Reply:
Done